src/Symfony/Component/DependencyInjection/Loader/schema/dic/services/services-1.0.xsd

```diff
@@ -279,4 +279,10 @@
       <xsd:pattern value="(%.+%|true|false)" />
     </xsd:restriction>
   </xsd:simpleType>
+
+  <xsd:simpleType name="autowire">
+    <xsd:restriction base="xsd:string">
+      <xsd:pattern value="(true|false|by-type|by-id)" />
```

Comment:
this does not allow parameters anymore, unlike ``boolean`` (see just above), so it is a BC break.